### ISSUE TITLE
Add the native gate(s) of TII quantum chips to qibo

### ISSUE THREAD

**Issue by wilkensJ** [qiboteam/qibo#847]
After the discussion with @ingoroth, @igres26  and @stavros11 we decided that it is highly useful to have gates which are native for specific platforms implemented in qibo.

The gate describing the native operation of the quantum chips used in TII is a RX rotation with an additional phase.
The matrix representation has to be looked up and implemented, dependent on the rotation angle and the phase (which is a virtual rotation around the Z axis)

**Comment by stavros11** [comment]
I was working out the math and went to add this gate, however I realized that we already have it as `GPI2`. Looking at [IonQ documentation](https://ionq.com/docs/getting-started-with-native-gates) it seems that they are applying single qubit gates exactly as we do. @igres26 @wilkensJ can you please confirm?

If yes, I will modify the qibolab transpiler to use this as native gate instead of U3.

**Comment by renatomello** [comment]
@stavros11 can this be closed?

**Comment by stavros11** [comment]
> [@stavros11](https://github.com/stavros11) can this be closed?

Yes, I think by now we all agree that GPI2 is our native and we already have it in Qibo, so I will close.
